cloudmock/aws/mockec2/images.go

```diff
@@ -46,18 +46,18 @@ func (m *MockEC2) DescribeImagesWithContext(aws.Context, *ec2.DescribeImagesInpu
 	panic("Not implemented")
 }
 
-func (m *MockEC2) DescribeImages(request *ec2.DescribeImagesInput) (*ec2.DescribeImagesOutput, error) {
+func (m *MockEC2) DescribeImagesPages(request *ec2.DescribeImagesInput, callback func(output *ec2.DescribeImagesOutput, b bool) bool) error {
```

Comment:
Can we use DescribeImagesPagesWithContext (rather than having to introduce it later)